### ISSUE TITLE
As a user I need to get the most necessary information to be able to understand the rules around Christmas tree harvest in Mt. Hood in addition to information that is helpful to know.

### ISSUE THREAD

**Issue by sadlerw** [nciinc/fs-intake-module#93]

**Comment by sadlerw** [comment]
Underneath the large green introductory header, include the following welcoming message "Your Christmas tree permit will allow you to cut a tree on the XX National Forest following the rules and regulations on your permit, you may cut a Christmas on the Forest, except as noted below in PROHIBITED information sections."  ADSprinkle

**Comment by sadlerw** [comment]
Blocked until we get info on which are rules and which are not.

**Comment by sadlerw** [comment]
[https://app.zenhub.com/files/106473503/f8b4c0ca-2ebc-4a90-93b7-558234c40bdb/download](https://app.zenhub.com/files/106473503/f8b4c0ca-2ebc-4a90-93b7-558234c40bdb/download)

**Comment by cameronwolf** [comment]
Screenshots of what was updated:
<img width="854" alt="screen shot 2017-11-17 at 5 13 31 pm" src="https://user-images.githubusercontent.com/7295948/32971396-13dcbb98-cbbb-11e7-9312-af973b4c31f7.png">
<img width="531" alt="screen shot 2017-11-17 at 5 14 02 pm" src="https://user-images.githubusercontent.com/7295948/32971397-13ed3ec8-cbbb-11e7-99cf-8b9adab92574.png">
<img width="807" alt="screen shot 2017-11-17 at 5 14 25 pm" src="https://user-images.githubusercontent.com/7295948/32971398-13facb1a-cbbb-11e7-843d-88a7ee904d1c.png">

**Comment by JaneZC** [comment]
Below are some recommended changes. (Note that I also have the remove places to try and remove hemlock species in here so those can be ignored.) 

[https://app.zenhub.com/files/106473503/fda7f030-2166-4f11-893a-59da6240d217/download](https://app.zenhub.com/files/106473503/fda7f030-2166-4f11-893a-59da6240d217/download)

Shortened Tree Cutting Info and removed After You Cut section
[https://app.zenhub.com/files/106473503/d4cab4da-1ff2-4121-bdda-a3ab5a789698/download](https://app.zenhub.com/files/106473503/d4cab4da-1ff2-4121-bdda-a3ab5a789698/download)

**Comment by bryanmcfadden** [comment]
Looks good to me.

**Comment by MelissaBraxton** [comment]
The addition of a rules section at the top of the page is a significant change to the hierarchy of information. We have evidence that people's key questions are when, where, and how to cut. While helping people understand how to be  safe stewards of the forest is also a priority,  I'd recommend putting the rules together in their own section, and prioritizing the information that helps people answer the when, where, and how questions at the top. 

Placing the rules in their own section would also negate the need for the red "!" throughout the page which is a bit jarring and would also be overwhelming if applied consistently (right now, there are some rules that do not have this indicator, which may confuse users, but applying it in every case would result in a lot of clutter). Also, the size of the "!" stays constant but the headers vary in size, so the "!" looks fine with H2 but overwhelms H4.  

The fact that there are things marked as rules that do not appear in the "Which rules you need to know" section is confusing--it implies that the other things marked as rules are rules you _don't_ need to know, or maybe not rules at all.

**Comment by ASprinkle** [comment]
@sadlerw @MelissaBraxton  I agree with Melissa's comments above. Order of information presented on the page should flow per user research and testing (something similar to this order); 
"When" information first followed by 
"Where", and "How" 
Trip Planning, 
Species Information, 
Contact Information.  
The "content flow" word doc that I shared earlier followed an old format from Sprint 2 - and I apologize to the team for that.
